styles.css

```diff
@@ -183,6 +183,10 @@ body{
     height: 240rem;
 }
 
+.container__iframe--image-component {
```

Comment:
Instead of manually adding height for iframe, try to use JS to calculate it. 

Please refer : https://stackoverflow.com/a/9163087/5615643

Reply:
Oh nice, I'll be using this while refactoring documentation code.
Thanks.